application/config/config.go

```diff
@@ -152,9 +155,8 @@ func IsDevelopment() bool {
 // New creates a configuration object with default values
 func New() *Config {
 	c := &Config{}
-	c.cliSettings = &CliSettings{
-		cliPath: filepath.Join(c.DefaultBinaryInstallPath(), filename.ExecutableName),
-	}
+	c.cliSettings = &CliSettings{}
+	c.cliSettings.SetPath("")
```

Comment:
Maybe better to have a `New()` function for CLI settings with all the default values